pkg/hooks/1.0.0/when_test.go

```diff
@@ -24,16 +24,22 @@ func TestNoMatch(t *testing.T) {
 
 func TestAlways(t *testing.T) {
 	config := &rspec.Spec{}
+	processStruct := &rspec.Process{
+		Args: []string{"/bin/sh", "a", "b"},
+	}
 	for _, always := range []bool{true, false} {
 		for _, or := range []bool{true, false} {
-			t.Run(fmt.Sprintf("always %t, or %t", always, or), func(t *testing.T) {
-				when := When{Always: &always, Or: or}
-				match, err := when.Match(config, map[string]string{}, false)
-				if err != nil {
-					t.Fatal(err)
-				}
-				assert.Equal(t, always, match)
-			})
+			for _, process := range []*rspec.Process{processStruct, nil} {
+				t.Run(fmt.Sprintf("always %t, or %t", always, or), func(t *testing.T) {
```

Comment:
This subtest name needs to be update to include the process-ness.  For one way to do that, see [my earlier suggestion][1].  Although you could also use something like:

```go
t.Run(fmt.Sprintf("always %t, or %t, process %t", always, or, process != nil), func(t *testing.T) {
```

[1]: https://github.com/containers/libpod/pull/1311#discussion_r211615507

Reply:
Fixed